overlays/keystone/board/sifive/hifive-unmatched/patches/uboot/board-sifive-prefer-mmc-boot.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/include/configs/sifive-unmatched.h b/include/configs/sifive-unmatched.h
```

Comment:
Add a number prefix to this patch to indicate the order in which you'd like the system to apply it